client/blocks/post-edit-button/index.jsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import { localize } from 'i18n-calypso';
+import { getEditURL } from 'lib/posts/utils';
+
+const PostEditButton = ( { post, site, size, translate } ) => {
+	const editUrl = getEditURL( post, site );
+	return (
+		<a className="post-edit-button" href={ editUrl }>
```

Comment:
should probably track clicks on this. are we doing that for the edit link in the menu?